### DISCUSSION TITLE
Geo Files能整合到软件里，用一个更新按扭来下载更新。

### DISCUSSION THREAD

**Discussion by Hunerk** [MetaCubeX/ClashMetaForAndroid#358] [General]
而不是现在这样导入，谢谢。

**Comment by Aladdinsane3** [comment]
你不是用flclash和用脚投票去了吗

**Reply by Hunerk** [reply]
> 你不是用flclash和用脚投票去了吗

我都用着的，flclash有个问题还没解决。我的节点在使用手机数据时，时不时会出现超时的情况。CMFA没这情况，我希望越做越好，
